lnwallet/wallet.go

```diff
@@ -325,6 +330,7 @@ func NewLightningWallet(cdb *channeldb.DB, notifier chainntnfs.ChainNotifier,
 	return &LightningWallet{
 		chainNotifier:    notifier,
 		Signer:           signer,
+		FeeEstimator:     StaticFeeEstimator{250, 6},
```

Comment:
This should likely just be passed into the wallet as an argument to the constructor. If we do this, it'll minimize the size of the diff once we switch out this static fee estimator for a real one.